docker-mixin/dashboards/docker/queries.libsonnet

```diff
@@ -0,0 +1,45 @@
+local host_matcher = 'job=~"$job", instance=~"$instance"';
```

Comment:
Quite common that query selectors depend on config (instead of hardcoded job=~"$job" it could be user provided label selector). Does this packaging provide a way to provide custom host_matcher from config?